webpack/components/DashboardSearch.js

```diff
@@ -167,6 +183,14 @@ class DashboardSearch extends Component {
                 <h2>Additonal Filters:</h2>
                 <input type='checkbox' className='form-check-input' name='most-recent-filter' id='most-recent-filter' checked={this.state.mostRecentFilter} onChange={() => this.toggleMostRecentFilter()} />
                 <label htmlFor="most-recent-filter">Most Recent Versions Only</label>
+                <div>
+                  <label htmlFor='content-since'>Content Changed Since</label>
```

Comment:
The htmlFor 'content-since' needs to match the id of an element on the page for label to be compliant. if there is a way to pass 'content-since' to SingleDatePicker to make it the id of the element it would then be compliant.